### DISCUSSION TITLE
Fall GE meetings

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#153] [Announcements]
Dear @delph-in/participants , for those interested in regular grammar engineering discussions, please fill out the poll for the upcoming semester (or at least quarter):

https://www.when2meet.com/?25947861-eINBM 

As usual, note that the specific dates should be ignored and treated simply as days of the odd and even weeks of the upcoming months!

**Comment by olzama** [comment]
Dear @delph-in/participants ,

Our first meeting is Monday September 16 at 18:30 Madrid time. 

Zoom: https://zoom.us/j/4768685765?pwd=NnZqQmx5QjcyNm94b1BuRUR3NUVkZz09

Please feel free to suggest some **agenda**. 

My personal agenda at this time includes:

0. Catch up after summer, goals for the upcoming semester
1. I could discuss my grant application having to do with using LLM to semi-automate grammar engineering process
2. @petterha has sent me a version of the Spanish grammar in the format that should be compatible with the HPSG-to-minimalist conversion that he presented at the summit. I would like to look at that also.

I do not yet have anything to say about the projects related to analyzing LLM's language syntax or using DELPH-IN for synthetic data; I haven't started any of that yet.

Please, feel free to suggest your agenda also.

**Comment by olzama** [comment]
Dear @delph-in/participants 
Some of us will be meeting Monday October 7 at 18:30 Madrid time:

https://zoom.us/j/4768685765?pwd=NnZqQmx5QjcyNm94b1BuRUR3NUVkZz09 

Please feel free to suggest agenda.

My agenda:

1) The llm syntactic footprint project; my colleagues from UDC have shared their data with me (this paper: https://assets-eu.researchsquare.com/files/rs-4077382/v1_covered_f0029c67-648a-49b6-b403-7e26416b5d53.pdf?c=1724688830) - for that, @fcbond would need to be attending. @fcbond if you confirm your attendance, I can try to invite my UDC colleagues to attend

2) If @emilymbender would be attending, the I know Zina Pozen wanted to attend to ask some questions about the Matrix and language collage grammars. Could you confirm one way or the other, @emilymbender ?

**Comment by emilymbender** [comment]
Yes, I can come this time!

**Comment by fcbond** [comment]
I should be able to attend.

On Fri, 4 Oct 2024 at 14:53, emilymbender ***@***.***> wrote:

> Yes, I can come this time!
>
> On Fri, Oct 4, 2024 at 5:26 AM Olga Zamaraeva ***@***.***>
> wrote:
>
> > Dear @delph-in/participants
> > <
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/teams/participants__;!!K-Hz7m0Vt54!gQwHcmFh9ERgLkutIzgXZ_YEHNms873sszTAInCZuYq1RmlylZBh0FW7kDmd8UfBROqxk85IL6ZtqssJQcz85CY$>
>
> > Some of us will be meeting Monday October 7 at 18:30 Madrid time:
> >
> > https://zoom.us/j/4768685765?pwd=NnZqQmx5QjcyNm94b1BuRUR3NUVkZz09
> > <
> https://urldefense.com/v3/__https://zoom.us/j/4768685765?pwd=NnZqQmx5QjcyNm94b1BuRUR3NUVkZz09__;!!K-Hz7m0Vt54!gQwHcmFh9ERgLkutIzgXZ_YEHNms873sszTAInCZuYq1RmlylZBh0FW7kDmd8UfBROqxk85IL6ZtqssJcngQYJE$>
>
> >
> > Please feel free to suggest agenda.
> >
> > My agenda:
> >
> > 1.
> >
> > The llm syntactic footprint project; my colleagues from UDC have
> > shared their data with me (this paper:
> >
> https://assets-eu.researchsquare.com/files/rs-4077382/v1_covered_f0029c67-648a-49b6-b403-7e26416b5d53.pdf?c=1724688830
> > <
> https://urldefense.com/v3/__https://assets-eu.researchsquare.com/files/rs-4077382/v1_covered_f0029c67-648a-49b6-b403-7e26416b5d53.pdf?c=1724688830__;!!K-Hz7m0Vt54!gQwHcmFh9ERgLkutIzgXZ_YEHNms873sszTAInCZuYq1RmlylZBh0FW7kDmd8UfBROqxk85IL6ZtqssJAEJqFIE$>)
>
> > - for that, @fcbond
> > <
> https://urldefense.com/v3/__https://github.com/fcbond__;!!K-Hz7m0Vt54!gQwHcmFh9ERgLkutIzgXZ_YEHNms873sszTAInCZuYq1RmlylZBh0FW7kDmd8UfBROqxk85IL6ZtqssJG7eYUVg$>
>
> > would need to be attending. @fcbond
> > <
> https://urldefense.com/v3/__https://github.com/fcbond__;!!K-Hz7m0Vt54!gQwHcmFh9ERgLkutIzgXZ_YEHNms873sszTAInCZuYq1RmlylZBh0FW7kDmd8UfBROqxk85IL6ZtqssJG7eYUVg$>
>
> > if you confirm your attendance, I can try to invite my UDC colleagues to
> > attend
> > 2.
> >
> > If @emilymbender
> > <
> https://urldefense.com/v3/__https://github.com/emilymbender__;!!K-Hz7m0Vt54!gQwHcmFh9ERgLkutIzgXZ_YEHNms873sszTAInCZuYq1RmlylZBh0FW7kDmd8UfBROqxk85IL6ZtqssJZdTPbhQ$>
>
> > would be attending, the I know Zina Pozen wanted to attend to ask some
> > questions about the Matrix and language collage grammars. Could you
> confirm
> > one way or the other, @emilymbender
> > <
> https://urldefense.com/v3/__https://github.com/emilymbender__;!!K-Hz7m0Vt54!gQwHcmFh9ERgLkutIzgXZ_YEHNms873sszTAInCZuYq1RmlylZBh0FW7kDmd8UfBROqxk85IL6ZtqssJZdTPbhQ$>
>
> > ?
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://urldefense.com/v3/__https://github.com/orgs/delph-in/discussions/153*discussioncomment-10843806__;Iw!!K-Hz7m0Vt54!gQwHcmFh9ERgLkutIzgXZ_YEHNms873sszTAInCZuYq1RmlylZBh0FW7kDmd8UfBROqxk85IL6ZtqssJ92iDuAE$>,
>
> > or unsubscribe
> > <
> https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AC5VF6V5EVRMXTHEECVYMPTZZ2CP5AVCNFSM6AAAAABMTQLWDWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAOBUGM4DANQ__;!!K-Hz7m0Vt54!gQwHcmFh9ERgLkutIzgXZ_YEHNms873sszTAInCZuYq1RmlylZBh0FW7kDmd8UfBROqxk85IL6ZtqssJI-u30Nw$>
>
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/153#discussioncomment-10844092>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRQRJNJUT5JNIG77UQTZZ2FTTAVCNFSM6AAAAABMTQLWDWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAOBUGQYDSMQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by danflick** [comment]
I'm sorry I'll have to miss today.  On a road trip this week to the
neighboring state of Arkansas with my brother for his birthday present.
Catch you all next time.

 Dan

**Comment by olzama** [comment]
Dear @delph-in/participants
Some of us will be meeting Friday October 18 at 18:30 Madrid time:

https://zoom.us/j/4768685765?pwd=NnZqQmx5QjcyNm94b1BuRUR3NUVkZz09

Please feel free to suggest agenda.

My agenda: Synthetic data, generation, and the possibility to produce data in UD format, somehow (so as to draw attention to it).

**Comment by danflick** [comment]
I'll be 30 minutes late, but will join you then if you're meeting.

 Dan

**Reply by olzama** [reply]
Great, thanks, @danflick ! I think I should be there in any case.

**Comment by olzama** [comment]
Some of us will be meeting today 18:30 Madrid time via zoom:

[https://zoom.us/j/4768685765?pwd=NnZqQmx5QjcyNm94b1BuRUR3NUVkZz09](https://www.google.com/url?q=https://zoom.us/j/4768685765?pwd%3DNnZqQmx5QjcyNm94b1BuRUR3NUVkZz09&sa=D&source=calendar&usd=2&usg=AOvVaw2taiKUt_prWz7-ANV-Z9Fd)

**Comment by olzama** [comment]
Some of us will be meeting today 18:30 Madrid time via zoom:

[https://zoom.us/j/4768685765?pwd=NnZqQmx5QjcyNm94b1BuRUR3NUVkZz09](https://www.google.com/url?q=https://zoom.us/j/4768685765?pwd%3DNnZqQmx5QjcyNm94b1BuRUR3NUVkZz09&sa=D&source=calendar&usd=2&usg=AOvVaw2taiKUt_prWz7-ANV-Z9Fd)
